src/record_layout.zig

```diff
@@ -116,18 +116,23 @@ const SysVContext = struct {
         fld_layout: TypeLayout,
     ) void {
         const bit_width = fld.bit_width.?;
-
         const ty_size_bits = fld_layout.size_bits;
         var ty_fld_algn_bits: u32 = fld_layout.field_alignment_bits;
 
-        // Some targets ignore the alignment of the underlying type when laying out
-        // non-zero-sized bit-fields. See test case 0072. On such targets, bit-fields never
-        // cross a storage boundary. See test case 0081.
-        if (comp.ignoreNonZeroSizedBitfieldTypeAlignment()) {
-            ty_fld_algn_bits = 1;
-        }
-
-        if (bit_width == 0) {
+        if (bit_width > 0) {
+            std.debug.assert(bit_width <= ty_size_bits); // TODO: currently checked in parser, move check here?
```

Comment:
Why would this check be moved?

Reply:
I meant to ask that in a comment on here; we can remove the todo and just leave the `checked in parser` part.